src/platform/packages/shared/presentation/presentation_publishing/interfaces/fetch/fetch.ts

```diff
@@ -48,6 +47,14 @@ import {
   type PublishesUnifiedSearch,
 } from './publishes_unified_search';
 
+// TODO Avoid redefining this here, fix circular dependency with presentation-containers
```

Comment:
Had to add this to get a green CI and wanted to get a second opinion on on how to ultimately resolve this circular dependency. In scope for this PR, or should we do something in a followup?

Reply:
I think it's better to tackle this in a follow up 🙇